src/moscot/base/problems/_mixins.py

```diff
@@ -458,80 +457,81 @@ def compute_feature_correlation(
         seed: Optional[int] = None,
         **kwargs: Any,
     ) -> pd.DataFrame:
-        """
-        Compute correlation of push or pull distribution with features.
+        """Compute correlation of push-forward or pull-back distribution with features.
+
+        Correlates a feature, e.g., counts of a gene, with probabilities of cells mapped to a set of cells such as
+        the push-forward or pull-back distributions.
 
-        Correlates a feature (e.g. counts of a gene) with probabilities of cells mapped to a set of cells, e.g.
-        a pull back or push forward distribution.
+        .. seealso::
+            - TODO: create and link an example
```

Comment:
@ArinaDanilina please open an issue for this and assign yourself

Reply:
#558